luaui/Widgets/cmd_share_unit.lua

```diff
@@ -5,7 +5,7 @@ function widget:GetInfo()
 		author = "SuperKitowiec",
 		date = "2024",
 		license = "GNU GPL, v2 or later",
-		version = 3.2,
+		version = 3.3,
```

Comment:
Version number only needs to be incremented once when merging to master, not multiple times within a single PR.
```suggestion
		version = 1.0,
```